setup.py

```diff
@@ -20,8 +40,14 @@
     url="https://github.com/lack/soundcraft-utils",
     license="MIT",
     packages=find_packages(),
+    # https://pypi.org/classifiers/
     classifiers=[
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
What's the benefit from defining these specific versions and not just leaving it as 'Python 3'?

Reply:
It's a shame there isn't a canonical way to record "This is the minimum supported python version".

Reply:
As a matter of fact, the `setup.py` line following the classifiers says exactly that:

```python
    python_requires=">=3.6",
```

I think there are different software components and services handling `python_requires` and `classifiers`.